ios/RNPiratechain.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,6 +2,6 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:RNZcash.xcodeproj">
+      location = "group:RNPiratechain.xcodeproj">
```

Comment:
One note: The XCode workspace doesn't actually do anything for native libraries, since CocoaPods does all the integration. If you look at edge-core-js, you will see that I simply add a dummy file instead of a complete project. Might be worth doing here too.